src/commands/mods-uninstall.ts

```diff
@@ -75,13 +75,26 @@ export default new Command("mods:uninstall <modInstanceId>")
     try {
       await modsApi.deleteInstance(projectId, instanceId);
       if (confirmedServiceAccountDeletion || options.force) {
-        await iam.deleteServiceAccount(projectId, instance.serviceAccountEmail);
-        utils.logLabeledBullet(
-          logPrefix,
-          `deleted service account ${clc.bold(instance.serviceAccountEmail)}`
+        const saDeletionRes = await iam.deleteServiceAccount(
+          projectId,
+          instance.serviceAccountEmail
         );
+        if (_.get(saDeletionRes, "body.error") && _.get(saDeletionRes, "body.error.code") !== 404) {
+          throw new FirebaseError("Unable to delete service account");
+        } else if (_.get(saDeletionRes, "body.error.code") === 404) {
```

Comment:
do you also have to check if `saDeleteRes` has a `body.error`?

It might be more readable to format this if/else statement as 
```
if (has body.error)
  if error === 404 {}
  else {}
else {success!}
```

Reply:
Seems like a good idea, especially since Bryan also found the current style hard to read